extension-workspace-references.md

```diff
@@ -157,6 +193,44 @@ interface WorkspaceSymbolParams {
     /**
      * Known properties about the symbol.
      */
-    symbol?: Partial<SymbolDescriptor>;
+    symbol?: Partial<SymbolIdentifier>;
 }
 ```
+
+### Workspace Packages Request
+
+This method returns metadata about the package(s) defined in a
+workspace and a list of dependencies for each package.
+
+This method is necessary to implement cross-repository jump-to-def
```

Comment:
This is probably too much about our specific use case for the specification. I think the usefulness of this information to language server clients is self-evident and we needn't spend too much text justifying it.